packages/eslint-plugin/tests/rules/naming-convention/naming-convention.test.ts

```diff
@@ -932,6 +932,26 @@ ruleTester.run('naming-convention', rule, {
         },
       ],
     },
+    {
+      code: `
+        import * as FooBar from 'foo_bar';
+        import fooBar from 'foo_bar';
+        import { default as fooBar } from 'foo_bar';
+        import { foo_bar } from 'foo_bar';
+      `,
+      parserOptions,
+      options: [
+        {
+          selector: ['import'],
+          format: ['PascalCase'],
+        },
+        {
+          selector: ['import'],
+          modifiers: ['default'],
+          format: ['camelCase'],
+        },
+      ],
+    },
```

Comment:
[Testing] https://typescript-eslint.io/contributing/local-development#granular-unit-tests - could you split these up please? Doing so will make the `// ❌ error` comments redundant.

We merged https://github.com/typescript-eslint/typescript-eslint/pull/7319 after this PR was sent, but we've been ramping up on enforcing that preference for a while.